apps/docs/src/app/platform/component-docs/platform-dynamic-page/platform-dynamic-page-examples/platform-dynamic-page-responsive-padding-example.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, ChangeDetectionStrategy } from '@angular/core';
+
+@Component({
+    selector: 'fdp-dynamic-page-responsive-padding-example',
+    templateUrl: './platform-dynamic-page-responsive-padding-example.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class PlatformDynamicPageResponsivePaddingExampleComponent {
+    onCollapseChange(event: Event): any {
+        console.log('collapse changed');
+    }
+
+    surveyClicked(event: Event): any {
```

Comment:
same as above

Reply:
added appropriate event type and return type as void.